src/cascadia/TerminalControl/Resources/en-US/Resources.resw

```diff
@@ -175,4 +175,10 @@
   <data name="TermControl_RendererRetryButton.Content" xml:space="preserve">
     <value>Resume</value>
   </data>
+  <data name="HowToOpenRun.Text" xml:space="preserve">
+    <value>ctrl+click to follow link</value>
+  </data>
+  <data name="HowToOpenRun.FontStyle" xml:space="preserve">
+    <value>Italic</value>

```

Comment:
ooh, we may want to lock this one or specify it in the xaml file -- the localizers may accidentally change "Italic" to.. whatever Italic is in 13 languages. 😁

Reply:
(probably just leave it in the xaml file -- it can have properties from both the uid/resw and the xaml)